sqlglot/dialects/mysql.py

```diff
@@ -12,6 +12,316 @@
 )
 from sqlglot.helper import seq_get
 from sqlglot.tokens import TokenType
+from sqlglot.trie import in_trie, new_trie
+
```

Comment:
I didn't want to pollute `sqlglot.expressions` with all these expression types, especially since they're not really transpile-able.

Reply:
I guess some of them are transpile-able... hm